package.json

```diff
@@ -19,13 +19,16 @@
     "xorshift": "^0.2.0"
```

Comment:
* L15 - is lodash a required runtime dependency? Can we avoid it?
* what is node-int64 used for?
* do we need assert as runtime dep?
* what is "request" module?